LEGO1/tgl/tgl.h

```diff
@@ -139,98 +123,69 @@ class Renderer : public Object {
 		unsigned long height
 	) = 0;
 	virtual Camera* CreateCamera() = 0;
-	virtual Light* CreateLight(LightType, double r, double g, double b) = 0;
-	virtual Group* CreateGroup(const Group* pParent = 0) = 0;
-
-	// pTextureCoordinates is pointer to array of vertexCount elements
-	//  (each element being two floats), or NULL
-	// pFaceData is faceCount tuples, each of format
-	//  [vertex1index, ... vertexNindex], where N = vertexPerFaceCount
-	virtual Mesh* CreateMesh(
-		unsigned long vertexCount,
-		const float (*pVertices)[3],
-		const float (*pTextureCoordinates)[2],
-		unsigned long faceCount,
-		unsigned long vertexPerFaceCount,
-		unsigned long* pFaceData
-	) = 0;
-	// pTextureCoordinates is pointer to array of vertexCount elements
-	//  (each element being two floats), or NULL
-	// pFaceData is:
-	//  [face1VertexCount face1Vertex1index, ... face1VertexMindex
-	//   face2VertexCount face2Vertex1index, ... face2VertexNindex
-	//   ...
-	//   0]
-	virtual Mesh* CreateMesh(
-		unsigned long vertexCount,
-		const float (*pVertices)[3],
-		const float (*pTextureCoordinates)[2],
-		unsigned long* pFaceData
-	) = 0;
+	virtual Light* CreateLight(LightType, float p_r, float p_g, float p_b) = 0;
+	virtual Group* CreateGroup(const Group* p_parent = 0) = 0;
+
+	// vtable+0x20
+	virtual Something* CreateSomething() = 0;
+	virtual Texture* CreateTexture() = 0;
 	virtual Texture* CreateTexture(
-		int width,
-		int height,
-		int bitsPerTexel,
-		const void* pTexels,
-		int pTexelsArePersistent,
-		int paletteEntryCount,
-		const PaletteEntry* pEntries
+		int p_width,
```

Comment:
For the utility libraries, the naming conventions we have in the rest of the code base don't apply:

https://github.com/isledecomp/isle/blob/master/CONTRIBUTING.md#naming-conventions

Instead, we want to retain the original naming scheme as far as it is available (even if inconsistent), so in this case we should leave all the parameter names as they were in the 1996 source. Thank you!